src/jquery/core.js

```diff
@@ -130,18 +130,18 @@ if ( jQueryVersionSince( "3.3.0" ) ) {
 
 	migratePatchAndWarnFunc( jQuery, "isNumeric", function( obj ) {
 
-			// As of jQuery 3.0, isNumeric is limited to
-			// strings and numbers (primitives or objects)
-			// that can be coerced to finite numbers (gh-2662)
-			var type = typeof obj;
-			return ( type === "number" || type === "string" ) &&
+		// As of jQuery 3.0, isNumeric is limited to
+		// strings and numbers (primitives or objects)
+		// that can be coerced to finite numbers (gh-2662)
+		var type = typeof obj;
+		return ( type === "number" || type === "string" ) &&
 
 				// parseFloat NaNs numeric-cast false positives ("")
 				// ...but misinterprets leading-number strings, e.g. hex literals ("0x...")
 				// subtraction forces infinities to NaN
 				!isNaN( obj - parseFloat( obj ) );
-		}, "isNumeric",
-		"jQuery.isNumeric() is deprecated"
+	}, "isNumeric",
+	"jQuery.isNumeric() is deprecated"
```

Comment:
This looks weird with the next line - the closing paren is at the same level as its last entry. Should we merge a few lines here or do something else to prevent this?

Reply:
I just ran prettier over it. Let me know what you think.